### DISCUSSION TITLE
看看我的驱动盘

### DISCUSSION THREAD

**Discussion by YukariZZZ** [share121/inter-knot#1162] [其他]
虽然很极品，但是一发才出
ps：我就是炫耀怎么滴
![WechatIMG247](https://github.com/user-attachments/assets/646906f6-3e36-4f94-898c-7a3fab415b10)

**Comment by Vicpest** [comment]
你不许玩绝区零！

**Comment by YukariZZZ** [comment]
新手池30抽狼叔，A级只给苍角，150发解决艾莲2+1，驱动盘刷了四个极品总共刷了不到30次。艾莲我天命！！！

**Comment by chaochao6667** [comment]
这游戏就不该免费给你玩

**Comment by 10kaihua** [comment]
或许我这个也不错呢（一天天的，这么点菲林全整驱动盘里了
![_cgi-bin_mmwebwx-bin_webwxgetmsgimg__ MsgID=4753553376497269491 skey=@crypt_b93f812c_914b766b53cf60073dd3e509b8b0ecf4 mmweb_appid=wx_webfilehelper](https://github.com/user-attachments/assets/5d16389a-9afe-4836-b3e5-497c44b15f66)
